docs/README-agent-install.md

```diff
@@ -27,6 +27,8 @@ This document describes Puppet agent installation and setup on Cisco Nexus switc
 
 See [References](#references) for alternative installation methods.
 
+**NOTE:** Agentless is the recommended mode of operation for the Cisco Nexus module. Agent installations are not recommended because the agent won't be supported beyond Puppet Enterprise 2018.1. For more information, see [Puppet Enterprise support lifecycle](https://puppet.com/misc/puppet-enterprise-lifecycle).
```

Comment:
I believe we need to clarify the context here.  This should be for `OAC` and `Native Bash` hosting options only.  Our guestshell environment runs Centos so the Centos agent will continue to be built right?

Reply:
@mikewiebe I've updated it, let me know what you think. Thanks!